graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/GroupBy.tsx

```diff
@@ -41,10 +41,13 @@ const GroupBy = ({ index }: Props) => {
       <FieldComponent index={index} fieldType={fieldType} />
       {fieldType === 'time' && (<Time index={index} />)}
       {fieldType === 'values' && (
-        <FormikFormGroup label="Limit" name={`groupBy.groupings.${index}.limit`} type="number" bsSize="small" />
+        <FormikFormGroup label="Limit"
+                         name={`groupBy.groupings.${index}.limit`}
+                         type="number"
+                         bsSize="small" />
       )}
     </Wrapper>
   );
 };
 
-export default GroupBy;
+export default React.memo(GroupBy);
```

Comment:
Tiny nitpick: I would prefer wrapping with `React.memo` at the place of the function declaration instead of wrapping the export, that makes it more visible. It is a bit different with `connect`, but it's usually a multi-line statement that is less subtle than `React.memo`, which is easy to overlook.